website/api/zkvm/optimization.md

```diff
@@ -421,9 +421,9 @@ cycle counts added.
 | LW rd,offset(rs1)   | Load Word                          | rd ← s32[rs1 + offset]                         | 1 if [paged-in](#paging) 1094 to 5130 otherwise |
```

Comment:
is `paged-in` correct in lines 419-423, or should those also be revised to say `dirty`?

Reply:
For loads, paged-in is correct.